test/framework/functional/vector/message_templates.go

```diff
@@ -0,0 +1,85 @@
+package vector
```

Comment:
Why does vector require its own set of message templates if our goal is to adhere to the same Viaq data model?

Reply:
there are some differences in the data model, as captured in the google docs sheet.